crates/turbopack-ecmascript/src/analyzer/imports.rs

```diff
@@ -176,18 +183,25 @@ impl ImportMap {
 struct Analyzer<'a> {
     data: &'a mut ImportMap,
     current_annotations: ImportAnnotations,
+    source: Option<Vc<Box<dyn Source>>>,
 }
 
 impl<'a> Analyzer<'a> {
     fn ensure_reference(
         &mut self,
         module_path: JsWord,
         imported_symbol: ImportedSymbol,
+        span: Span,
```

Comment:
nitpick: Typically `Span` comes first if there's only one `Span` in the parameter list in SWC.
It's for SWC, not turbopack, but just FYI